model/registry/faas.yaml

```diff
@@ -0,0 +1,179 @@
+groups:
+  - id: registry.faas
+    brief: FaaS attributes
+    type: attribute_group
+    prefix: faas
+    attributes:
+      - id: name
```

Comment:
Hum these was used as a resource attribute before. With the problem we faced in the tooling when generating code from it, I think we need to mark this as type: resource still. fyi @AlexanderWert

Reply:
I think we should block PRs that move a namespace to the registry, that contains both semantic attributes and resource attributes until we found a proper solution for this in the tooling.